cx_Freeze/macdist.py

```diff
@@ -133,7 +133,6 @@ def setRelativeReferencePaths(self):
             files.extend([os.path.join(root, f).replace(self.binDir + "/", "")
                           for f in dir_files])
         for fileName in files:
-
```

Comment:
Please don't make arbitrary white space changes in pull requests. Can you revert this and the other one?